### PR TITLE
Quick fmask style

### PR DESCRIPTION
**Pull request by GypsyBojangles**
Feel free to suggest alternate colours
(previous PR updated prod instead of dev)

### CODE REVIEW

**Comment by MatthewJA** [comment]
@GypsyBojangles how would you feel about this scheme, which @Kirill888, @robbibt and I suggested deep in a Slack thread?

#C2C1C0 Cloud
#4B4B37 Shadow
#556E8B Water
#A0C591 Clear
#E0EEFF Snow

**Comment by pindge** [comment]
also need to update inventory style count.

File: dev/services/wms/ows_refactored/c3/style_c3_cfg.py [hunk 1]
```diff
@@ -453,6 +453,22 @@
     ],
 }
 
+style_c3_fmask = {
+    "name": "fmask",
+    "title": "FMASK - Pixel Classification",
+    "abstract": "Fmask (Function of mask) is used for automated clouds, cloud shadows, snow, and water masking for Landsats 4-8 and Sentinel 2 images.",
+    "include_in_feature_info": False,
+    "needed_bands": ["oa_fmask"],
```
Comment (pindge) [inline dev/services/wms/ows_refactored/c3/style_c3_cfg.py:461]: add `oa_fmask` to commonbands

**Review by pindge — CHANGES_REQUESTED** [review]

File: dev/services/wms/ows_refactored/c3/style_c3_cfg.py [hunk 2]
```diff
@@ -453,6 +453,22 @@
...
+    "color_ramp": [
+        {"value": 0, "color": "#ff0000", "alpha": 0.0},  # red - nodata - what could go wrong if we make it red?
+        {"value": 1, "color": "#A0C591"},  # clear/valid
+        {"value": 2, "color": "#C2C1C0"},  # cloud
+        {"value": 3, "color": "#4B4B37"},  # shadow
+        {"value": 4, "color": "#E0EEFF"},  # snow
+        {"value": 5, "color": "#556E8B"},  # water
+    ],
+}
+
```
Comment (SpacemanPaul) [inline dev/services/wms/ows_refactored/c3/style_c3_cfg.py:471]: This should use a colour map, not a ramp.  See https://datacube-ows.readthedocs.io/en/latest/style_howto_color_map.html

**Review by SpacemanPaul — CHANGES_REQUESTED** [review]

File: dev/services/wms/ows_refactored/c3/style_c3_cfg.py (hunk 1 above)
Comment (GypsyBojangles) [reply dev/services/wms/ows_refactored/c3/style_c3_cfg.py:461]: https://github.com/GeoscienceAustralia/dea-config/blob/9d76945b6edc75383543b817fe8dcc891ebe2828/dev/services/wms/ows_refactored/c3/ows_c3_cfg.py#L13 ?

Comment (GypsyBojangles) [reply dev/services/wms/ows_refactored/c3/style_c3_cfg.py:461]: isn't that it?

**Comment by pindge** [comment]
the reason ci is failing is because the 4 layers effected by this new style is not loading correctly.

File: dev/services/wms/ows_refactored/c3/style_c3_cfg.py (hunk 2 above)
Comment (GypsyBojangles) [reply dev/services/wms/ows_refactored/c3/style_c3_cfg.py:471]: thank you!

Comment (SpacemanPaul) [reply dev/services/wms/ows_refactored/c3/style_c3_cfg.py:471]: e.g.

```
    "value_map": {
         "oa_fmask": [
              {
                  "title": "No Data",
                  "abstract": "",
                  "values": [0],
                  "color": "#FF0000",
              }, ....
         ]
    }
```

File: dev/services/wms/ows_refactored/c3/style_c3_cfg.py (hunk 1 above)
Comment (pindge) [reply dev/services/wms/ows_refactored/c3/style_c3_cfg.py:461]: yes ok

File: dev/services/wms/ows_refactored/c3/style_c3_cfg.py (hunk 2 above)
Comment (GypsyBojangles) [reply dev/services/wms/ows_refactored/c3/style_c3_cfg.py:516]: ok, I have implemented one, does that look right?

**Review by pindge — APPROVED** [review]
Yay, CI is passing

**Review by SpacemanPaul — APPROVED** [review]
Looks good to me.

**Pull request merged**